src/main/resources/anniv5Resources/localization/eng/contentcreatorpack/Cardstrings.json

```
@@ -0,0 +1,64 @@
+{
+  "${ModID}:OlexasCloak": {
+    "NAME": "Olexa's Cloak",
+    "DESCRIPTION": "Retain. NL ${ModID}:Hoard !hoard!: Gain !B! Block. NL stslib:Persist !stslib:persist! ."
+  },
+  "${ModID}:HuttsGamble": {
+    "NAME": "Hutts' Gamble",
+    "DESCRIPTION": "If you don't receive unblocked attack damage this turn, gain [E] and draw 1 card next turn.",
+    "UPGRADE_DESCRIPTION": "If you don't receive unblocked attack damage this turn, gain [E] [E] and draw 1 card next turn."
+  },
+  "${ModID}:Tricky": {
+    "NAME": "Tricky",
+    "DESCRIPTION": "Deal !D! damage to ALL enemies. NL Deals !M! additional damage for each card drawn this turn."
+  },
+  "${ModID}:SneakyteakStrike": {
+    "NAME": "Sneakyteak Strike",
+    "DESCRIPTION": "Deal !D! damage. NL Put a *Darkside *Slap on top of your draw pile.",
+    "UPGRADE_DESCRIPTION": "Deal !D! damage. NL Put a *Darkside *Slap+ on top of your draw pile."
+  },
+  "${ModID}:DarksideSlap": {
+    "NAME": "Darkside Slap",
+    "DESCRIPTION": "${ModID}:MEGAthereal. NL Deal !D! damage."
+  },
+  "${ModID}:Wanderbots": {
+    "NAME": "Wanderbots",
+    "DESCRIPTION": "Gain !B! Block. NL Channel a ${ModID}:Wanderbot.",
+    "UPGRADE_DESCRIPTION": "${ModID}:Inscribed. NL Gain !B! Block. NL Channel a ${ModID}:Wanderbot."
+  },
+  "${ModID}:Retromation": {
+    "NAME": "Retromation",
+    "DESCRIPTION": "At the start of your turn, gain a card from a Pack not available to your draft. NL It gains Ethereal."
```

Minor and probably debatable, but I would phrase this as *in* your draft.
```suggestion
    "DESCRIPTION": "At the start of your turn, gain a card from a Pack not available in your draft. NL It gains Ethereal."
```